docs/zkapps/tutorials/03-deploying-to-a-network.mdx

```diff
@@ -256,17 +246,24 @@ A deploy alias consists of:
 1. Review and confirm the details of the transaction: 
   
   ```text
-  ✖ Confirm to send transaction
-
-    ┌─────────────────┬──────────────────────────────────────────────────┐
-    │ Deploy Alias    │ berkeley                                         │
-    ├─────────────────┼──────────────────────────────────────────────────┤
-    │ Fee-Payer Alias │ testing-feepayer                                 │
-    ├─────────────────┼──────────────────────────────────────────────────┤
-    │ URL             │ https://proxy.berkeley.minaexplorer.com/graphql  │
-    ├─────────────────┼──────────────────────────────────────────────────┤
-    │ Smart Contract  │ Square                                           │
-    └─────────────────┴──────────────────────────────────────────────────┘
+  ✔ Confirm to send transaction
+
+    |-----------------|-------------------------------------------------|
+    | Deploy alias    | berkeley                                        |
+    |-----------------|-------------------------------------------------|
+    | Network kind    | testnet                                         |
+    |-----------------|-------------------------------------------------|
+    | URL             | https://proxy.berkeley.minaexplorer.com/graphql |
+    |-----------------|-------------------------------------------------|
+    | Fee payer       | Alias         : 03-deploy                       |
+    |                 | Account       : B62...                          |
+    |-----------------|-------------------------------------------------|
+    | zkApp           | Smart contract: Square                          |
```

Comment:
I generally like this table view change but it may introduce some clutter/complexity in the CLI. I will add my comments there for further discussion.

Reply:
Further discussion is moved to the https://github.com/o1-labs/zkapp-cli/issues/633.